lease-kubernetes/src/main/scala/akka/coordination/lease/kubernetes/internal/KubernetesApiImpl.scala

```diff
@@ -227,7 +227,7 @@ PUTs must contain resourceVersions. Response:
   }
 
   private def pathForLease(name: String): Uri.Path =
-    Uri.Path.Empty / "apis" / "akka.io" / "v1" / "namespaces" / namespace / "leases" / name
+    Uri.Path.Empty / "apis" / "akka.io" / "v1" / "namespaces" / namespace / "leases" / name.toLowerCase
```

Comment:
Aside from lower-casing, perhaps we should remove more invalid characters from the name?

Reply:
Hows that?